pkg/archive/changes_unix.go

```diff
@@ -25,3 +26,14 @@ func statDifferent(oldStat *system.StatT, newStat *system.StatT) bool {
 func (info *FileInfo) isDir() bool {
 	return info.parent == nil || info.stat.Mode()&syscall.S_IFDIR != 0
 }
+
+func getIno(fi os.FileInfo) (inode uint64) {
+	s, _ := fi.Sys().(*syscall.Stat_t)
```

Comment:
can be `return uint64(fi.Sys().(*syscall.Stat_t).Ino`. It's already uint64 and you don't need named return argument.